src/main/java/rpgcharacters/RpgCharacters.java

```diff
@@ -78,7 +76,40 @@ public static void main(String[] args) {
             jc.usage();
             return;
         }
+        //Create Roles
+        try {
+            Statement s = conn.createStatement();
+
+            String userView = "CREATE VIEW userView AS "
+                            + "SELECT * "
+                            + "FROM party, character;";
```

Comment:
Also I think the creation of the roles should happen in the create tables command